packages/api/src/lockfileSchema.ts

```diff
@@ -0,0 +1,84 @@
+import type { FromSchema } from '@readme/api-core/lib';
+
+const lockfileApiSchema = {
+  type: 'object',
+  required: ['createdAt', 'identifier', 'installerVersion', 'integrity'],
+  properties: {
+    createdAt: {
+      type: 'string',
+      format: 'date-time',
+      description: 'The date that this SDK was installed.',
+      examples: ['2023-10-19T20:35:39.268Z'],
+    },
+    identifier: {
+      type: 'string',
+      description:
+        "A unique identifier of the API. This'll be used to do imports on `@api/<identifier>` and also where the SDK code will be located in `.api/apis/<identifier>`",
+      examples: ['petstore'],
+    },
+    installerVersion: {
+      type: 'string',
+      description: 'The version of `api` that was used to install this SDK.',
+      examples: ['7.0.0'],
+    },
+    integrity: {
+      type: 'string',
+      description:
+        'An integrity hash that will be used to determine on `npx api update` calls if the API has changed since the SDK was last generated.',
+      examples: ['sha512-ld+djZk8uRWmzXC+JYla1PTBScg0NjP/8x9vOOKRW+DuJ3NNMRjrpfbY7T77Jgnc87dZZsU49robbQfYe3ukug=='],
+    },
+    language: {
+      type: 'string',
+      description: 'The language that this SDK was generated for.',
+      default: 'js',
+      enum: ['js'],
```

Comment:
Now that this isn't JSON we can use the `SupportedLanguages` const we've got:

```suggestion
      default: SupportedLanguages.JS,
      enum: Object.values(SupportedLanguages),
```

![Screen Shot 2023-10-20 at 10 08 15 AM](https://github.com/readmeio/api/assets/33762/23190897-6f3e-4945-808f-8f9be64a51a9)

Reply:
oooh smart!